cypress/support/step_definitions/dynamicCalls.js

```diff
@@ -348,6 +348,20 @@ Given(
  * Given 'third party app is launched' on 'auth' page
  */
 Given(/'(.+)' on '(.+)' page/, (validationObjectKey, page) => {
+  const requestMap = {
+    method: CONSTANTS.REQUEST_OVERRIDE_CALLS.GETAPPSTATE,
+    params: Cypress.env(CONSTANTS.CURRENT_APP_ID),
+  };
+
+  // Sending the request to the platform to retrieve the app state.
+  cy.sendMessagetoPlatforms(requestMap).then((response) => {
+    if (response === 'FOREGROUND') {
+      fireLog.info(`State validation successful: Current state is '${response}' as expected`);
+    } else {
+      fireLog.fail(`Unexpected state: ${response}. Expected to be FOREGROUND`);
+    }
+  });
```

Comment:
where is the documentation on fcs.getAppState ?

Reply:
added